test/tests/functional/pbs_ralter.py

```diff
@@ -2287,6 +2287,35 @@ def test_alter_standing_resv_future_occrs(self):
         rid2, start, end = self.submit_and_confirm_reservation(
             3000, 1800, select="2:ncpus=4")
 
+    def test_alter_start_standing_resv_future_occrs(self):
+        """
+        Test that when duration of a confirmed standing reservation is
+        altered, only the upcoming occurence changes and not all occurences
+        are modified.
+        """
+
+        duration = 180
+        offset = 30
```

Comment:
Since you aren't waiting for these reservations to start or end, can you make the offset and duration larger?